source/administration/config-settings.rst

```diff
@@ -2235,3 +2235,19 @@ Standard setting for OAuth to determine the scope of information shared with OAu
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | This feature's ``config.json`` setting is ``"Scope": "User.Read"`` with string input                                                                                 |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+________
+
+Analytics Settings
+```````````````````````````
+*Available in Enterprise Edition E10 and higher*
+
+Maximum Users for Statistics
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This underline is too short for the title. The Sphinx processor will ignore it, although the GitHub .rst processor does not.